packages/driver/src/cypress/stack_utils.ts

```diff
@@ -142,13 +142,25 @@ const getCodeFrameFromSource = (sourceCode, { line, column: originalColumn, rela
     line,
     column,
     originalFile: relativeFile,
-    relativeFile,
+    relativeFile: getRelativePathFromRoot(relativeFile, absoluteFile),
     absoluteFile,
     frame,
     language: getLanguageFromExtension(relativeFile),
   }
 }
 
+const getRelativePathFromRoot = (relativeFile, absoluteFile) => {
+  // at this point relativeFile is relative to the cypress config
+  // we need it to be relative to the repo root, which is different for monorepos
+  const repoRoot = Cypress.config('repoRoot')
+
+  if (absoluteFile?.startsWith(`${repoRoot}/`)) {
+    return absoluteFile.replace(`${repoRoot}/`, '')
+  }
```

Comment:
Just as a sanity check, is this working on windows? I am not sure if stack traces are always using posix (so `/`). I am pretty sure they are, but good to check, since we aren't running these tests on windows CI until they merge. I'm thinking we either

- manually run these tests on windows
- enable windows CI

Not sure which is faster/easier, up to you.

Reply:
Great callout! Just enabled Windows CI, fingers crossed 🤞

Reply:
Yay it's passing on Windows! 🪟

Reply:
Ahhhh @tbiethman pointed out that the windows executors do not run driver tests. I'll test this manually on a Windows VM later today!

Reply:
@tbiethman is a national treasure. This was broken on windows and has now been fixed. THANK YOU!